### DISCUSSION TITLE
Open Alipay for sponsor.

### DISCUSSION THREAD

**Discussion by xieyezi** [babel/babel#13318] [Ideas]
Hope Open Alipay for donation, there are many developer come from china want to sponsor babel.

We are all Hope babel will be better more!💪💪💪

**Comment by nicolo-ribaudo** [comment]
This is a very good idea! We currently use OpenCollective or GitHub Sponsors to receive donations, but I admit I don't know if they work in China and if they support the other payment methods used in China.

OpenCollective currently supports Paypal (a Google search tells me that it operates in China, but I don't know how common it is), but they are looking into supporting other payment systems (general issue: https://github.com/opencollective/opencollective/issues/2274, Alipay/WeChar: https://github.com/opencollective/opencollective/issues/2207).

I don't think we can hook an Alipay account to OpenCollective until it's officially supported, but I'll leave a comment on their issue letting them know that we are interested in that feature!

**Comment by nicolo-ribaudo** [comment]
Update: OpenCollective is working on Alipay support! https://github.com/opencollective/opencollective/issues/2207#issuecomment-852146658
